### DISCUSSION TITLE
Can't save meal plans

### DISCUSSION THREAD

**Discussion by bee-san** [TandoorRecipes/recipes#2571] [General]
Hi, I can't save a meal plan.
<img width="798" alt="Screenshot 2023-08-07 at 11 41 12" src="https://github.com/TandoorRecipes/recipes/assets/10378052/72138c05-3438-4534-b3dc-a3c5d9f74207">

When I click "save" nothing happens and there's nothing in the console. I cannot see any issues in the docker logs either.

Anyone come across something like this before?

**Comment by bee-san** [comment]
Ah, when I press "delete" I get:

```
meal_plan_view.js:1 Uncaught (in promise) TypeError: e.$createElement is not a function
    at _.makeStandardToast (meal_plan_view.js:1:59074)
    at meal_plan_view.js:1:40936
```

**Reply by smilerz** [reply]
how did you install Tandoor?  What version are you running?

**Comment by vabene1111** [comment]
the meal type is missing .. i really need to add a proper error message there ...

**Reply by bee-san** [reply]
That worked, thanks!
